main.cpp

```diff
@@ -108,7 +108,26 @@ struct CarWash
     You'll need to insert the Person struct from the video in the space below.
  */
 
+struct Person
+{
+	int age;
+	int height;
+	float hairLength;
+	float GPA;
+	unsigned int SATScore;
+	int distanceTraveled;
+
+	void run(int howFast, bool startWithLeftFoot);
 
+	struct Foot
+	{
+		void stepForward();
+		int stepSize();
+	};
+
+	Foot leftFoot;
+	Foot rightFoot;
+};
```

Comment:
- [ ] revise, indentation: see coding standard for what indentation settings to use.